### ISSUE TITLE
Training how to choose # of training set and # of validation set

### ISSUE THREAD

**Issue by HenryJunW** [yulunzhang/RCAN#24]
Wonderful work! I got a question, if I want to train my own dataset, like there are 3000 images in the training set in total, the question is how to set the # of training images and # of validation images? For example, in your provided code, for the div2k dataset, --n_train = 800, and --n_val = 5. Is there any underlying reason to choose those two numbers? Thanks

**Comment by yulunzhang** [comment]
Hi,

If you have total 3000 images in training set. You can set --n_train = 2995, --offset_val = 2995, and --n_val = 5. It means you use 2995 for training and 5 images for validation. The reason why use --n_val = 5 is just to save time. You can choose validation number as you like.

More details, see RCAN/RCAN_TrainCode/code/data/div2k.py Line 20 - 25
